SkinDarkVector.php

```diff
@@ -35,8 +35,10 @@ class SkinDarkVector extends SkinTemplate {
 	 */
 	private $darkvectorConfig;
 
-	public function __construct() {
+	public function __construct( $options = [] ) {
+		$options['bodyOnly'] = true;
```

Comment:
It looks like this option does not exist in 1.38 (which is current stable release):
https://github.com/wikimedia/mediawiki/blob/REL1_38/includes/skins/Skin.php#L2630
https://github.com/wikimedia/mediawiki/blob/master/includes/skins/Skin.php#L2319

So if user will install a patched version on 1.38, it will fail?

Reply:
Created new branch mw-1.39 and set it as pull request target, to avoid breaking backward compatibility with MW <1.38